app/views/movies/show.html.erb

```diff
@@ -0,0 +1,25 @@
+<%= button_to "New Viewing Party", "/viewing_parties/new?movie_id=#{@movie.id}", method: :get %>
+
+<h3><%= @movie.title %></h3>
+<p>Vote Average: <%= @movie.vote_avg %></p>
+<p>Runtime: <%= @movie.runtime %> minutes</p>
+<p>Summary: <%= @movie.summary %></p>
+<p>Genres:</p>
+<ul>
+<% @movie.genres.each do |genre| %>
+  <li><%= genre %></li>
+<% end %>
+</ul>
+<p>Cast (top 10):</p>
+<ul>
+<% @movie.first_ten_cast_members.each do |member| %>
+  <li><%= member.name %> -- as <%= member.character %></li>
+<% end %>
+</ul>
+<h4>Reviews(<%= @movie.reviews.count %>)</h4>
```

Comment:
`count` might be a _slightly_ better fit as a PORO method so we're limiting the logic that happens in the view, and we may want to change it to `size` or `length` for optimization